hil-test/tests/spi_full_duplex.rs

```diff
@@ -2,9 +2,8 @@
 //!
 //! Folowing pins are used:
 //! SCLK    GPIO0
-//! MISO    GPIO2 / GPIO9 (esp32s2 and esp32s3)
-//! MOSI    GPIO3 / GPIO10 (esp32s2 and esp32s3)
-//! CS      GPIO8
+//! MISO    GPIO2 / GPIO9 / GPIO26  (esp32s2 / esp32s3 / esp32)
+//! MOSI    GPIO3 / GPIO10 / GPIO27 (esp32s2 / esp32s3 / esp32)
```

Comment:
Same as before, maybe we should use the same way to present the pins being used as in other tests, eg: https://github.com/esp-rs/esp-hal/pull/1977/files#diff-7f3ecf58e488118925d4a5bdc6021bc63490beb6161d1e2f0a0940c0aad80aacR3-R5

Reply:
There are multiple other tests with the same error